stories/Atom/Cards/Publication-thumbnail/publication-thumbnail.scss

```diff
@@ -1,9 +1,9 @@
 @import '../../../assets/scss/mixins';
 
 /* publication_thumbnail */
-$colors: ('yellow': $color-yellow, 'red': $color-red, 'green': $color-green, 'azure': $color-azure);
+$colors: ('yellow': $color-yellow, 'red': $color-red, 'green': $color-green, 'blue': $color-azure);
 
-.publication-thumbnail__image {
+div.publication-thumbnail__image {
```

Comment:
Hey @harishkumar-srijan  adding div tag is not a correct way. if in future tag is changed then this css will not work.
Same for below code